### PR TITLE
Add Status on the AuthRealm

### PR DESCRIPTION
**Pull request by itdove**

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -30,8 +30,8 @@ spec:
         - -v 5
         env:
         - name: RELATED_IMAGE_DEX_OPERATOR
-          value: "quay.io/identitatem/dex-operator@sha256:b5f33d7939f765cabfa866b9bc74d1071c633bc308ed27288f7b84999f459a81"
-        image: quay.io/identitatem/idp-mgmt-operator@sha256:e499da1e4a2ddf6c4845bab8c9e21978487132c8b1dae5082dbe6739a5b15caf
+          value: "quay.io/identitatem/dex-operator@sha256:32829b387b9a7c490dcf67461b72597c6fa22c23ea5997bc8a53bcd11e6c043f"
+        image: quay.io/identitatem/idp-mgmt-operator@sha256:bf101fed1de2da84e200091748267faff1ce635bc73d6c541eea562719ba30c5
```
Comment (chrisahl) [inline config/manager/manager.yaml:34]: assume you only want dex-operator pinned to specific version?

File: go.sum [hunk 2]
```diff
@@ -1098,6 +1096,8 @@ github.com/inconshreveable/mousetrap v1.0.0 h1:Z8tu5sraLXCXIcARxBp/8cbvlwVa7Z1NH
 github.com/inconshreveable/mousetrap v1.0.0/go.mod h1:PxqpIevigyE2G7u3NXJIT2ANytuPF1OarO4DADm73n8=
 github.com/influxdata/influxdb v0.0.0-20190411212539-d24b7ba8c4c4/go.mod h1:qZna6X/4elxqT3yI9iZYdZrWWdeFOOprn86kgg4+IzY=
 github.com/influxdata/influxdb v1.7.7/go.mod h1:qZna6X/4elxqT3yI9iZYdZrWWdeFOOprn86kgg4+IzY=
+github.com/itdove/idp-client-api v0.0.0-20211005141937-61a3da19b015 h1:oParYSk6HIlFbKmRKoYr/L+wtSdvpJJycv/d/zxpDpQ=
```
Comment (chrisahl) [inline go.sum:1099]: Don't we need to merge idp-client-api first and not pull from itdove?

File: go.mod [hunk 3]
```diff
@@ -27,6 +27,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.9.6
 )
 
+replace github.com/identitatem/idp-client-api => github.com/itdove/idp-client-api v0.0.0-20211005141937-61a3da19b015
```
Comment (leena-jawale) [inline go.mod:30]: forgot to change ?

File: config/manager/manager.yaml (hunk 1 above)
Comment (itdove) [reply config/manager/manager.yaml:34]: ok.

File: go.sum (hunk 2 above)
Comment (itdove) [reply go.sum:1099]: yes sorry... to tired

File: go.mod (hunk 3 above)
Comment (itdove) [reply go.mod:30]: same as above

**Review by chrisahl — APPROVED** [review]

**Pull request merged**